### PR TITLE
Header and love to images

### PR DESCRIPTION
**Pull request by MatthysDucrocq**

### CODE REVIEW

File: src/stylesheets/article.css [hunk 1]
```diff
@@ -6,3 +6,10 @@
 .article-preview:hover h3 {
   text-decoration: underline;
 }
+.article-preview:hover .img {
```
Comment (flexbox) [inline src/stylesheets/article.css:9]:
```suggestion
.item-episode:hover .img,
.article-preview:hover .img {
  transform: scale(1.05);
}
```

But transition is missing 

```
  transition: ease-in-out 300ms;
```

have a look at https://codepen.io/dwinnbrown/pen/kXEEAa

File: src/stylesheets/article.css [hunk 2]
```diff
@@ -6,3 +6,32 @@
...
+  width: 100%;
+}
+.article-preview .img {
+  height: 100%;
+  width: 100%;
+  background-size: cover;
+  background-repeat: no-repeat;
+  -webkit-transition: all 0.5s;
+  -moz-transition: all 0.5s;
+  -o-transition: all 0.5s;
```
Comment (flexbox) [inline src/stylesheets/article.css:20]: you don't need these because we have `autoprefixer`
https://github.com/postcss/autoprefixer

Comment (MatthysDucrocq) [reply src/stylesheets/article.css]: fix

Comment (flexbox) [reply src/stylesheets/article.css]: You can just add 
```css
transition: ease-in-out 300ms;
  ```
  not delete everything :)

**Review by flexbox — APPROVED** [review]

**Pull request merged**